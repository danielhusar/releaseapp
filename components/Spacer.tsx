import styled from './Styled'

const Spacer = styled.div<{ size: number }>`
  height: ${({ size }) => size * 10}px;
`

export default Spacer
