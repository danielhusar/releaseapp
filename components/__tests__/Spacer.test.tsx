import React from 'react'
import { render } from './helpers'
import Spacer from '../Spacer'

describe('Spacer', () => {
  it('renders', () => {
    const { container } = render(<Spacer size={5} />)
    expect(container).toMatchSnapshot()
  })
})
