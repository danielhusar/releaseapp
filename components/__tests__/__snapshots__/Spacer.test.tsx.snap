// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spacer renders 1`] = `
.emotion-0 {
  height: 50px;
}

<div>
  <div
    class="emotion-0"
    size="5"
  />
</div>
`;
